doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataModeller/sections/FirstSteps-section.adoc

```diff
@@ -14,7 +14,7 @@ To start the creation of a data model inside a project, take the following steps
 image::Workbench/Authoring/DataModeller/authoring.jpg[align="center"]
 
 +
-. Open the Data Modeller tool by clicking on a Data Object file, or using the "Create New Asset -> Data Object" menu option. Set Data Object name to "PurchaseOrder" and click Ok.
+. Open the Data Modeller tool by clicking on a Data Object file, or using the "Add Asset -> Data Object" menu option. Set Data Object name to "PurchaseOrder" and click Ok.
```

Comment:
by clicking, not clicking on. Delete comma before or. This is two steps. Bold OK.

Reply:
ah.. These last topics are not mine.